doc/index.rst

```diff
@@ -64,7 +64,6 @@ Documentation
 * :doc:`introduction`
 * :doc:`installation`
 * :doc:`quick_start`
-* :doc:`examples`
```

Comment:
What would you think of the order

- getting started
- examples
- user guid
- help and references

?

Reply:
yes, why not? that is probably more beginner friendly

Reply:
Yes, that's what I thought